config/locales/headings/en.yml

```
@@ -70,6 +70,6 @@ en:
     verify_email: Check your email
     verify_personal_key: Verify your personal key
     webauthn_platform_setup:
-      new: Use your device
+      new: Add Face or touch unlock
```

I think this is meant to be lowercase

```suggestion
      new: Add face or touch unlock
```